core/page-assets/loader.php

```diff
@@ -41,21 +41,25 @@ private function init_styles(): array {
 				'src' => $this->get_css_assets_url( 'animations', 'assets/lib/animations/', true ),
 				'version' => ELEMENTOR_VERSION,
 				'dependencies' => [],
+				'loadInEditor' => false,
 			],
 			'e-shapes' => [
 				'src' => $this->get_css_assets_url( 'shapes', 'assets/css/conditionals/' ),
 				'version' => ELEMENTOR_VERSION,
 				'dependencies' => [],
+				'loadInEditor' => true,
 			],
 			'e-swiper' => [
 				'src' => $this->get_css_assets_url( 'e-swiper', 'assets/css/conditionals/' ),
 				'version' => ELEMENTOR_VERSION,
 				'dependencies' => [ 'swiper' ],
+				'loadInEditor' => true,
 			],
 			'swiper' => [
 				'src' => $this->get_css_assets_url( 'swiper', $this->getSwiperPath() ),
 				'version' => $this->getSwiperVersion(),
 				'dependencies' => [],
+				'loadInEditor' => true,
```

Comment:
should we name it in a way `load_in_editor`? 
camel case is not WP/Elementor-way

Reply:
@mykytamurzin Updated.